rosidl_runtime_c/include/rosidl_runtime_c/action_type_support_struct.h

```diff
@@ -27,6 +27,8 @@ extern "C"
 
 typedef struct rosidl_action_type_support_t rosidl_action_type_support_t;
 
+/// Contains rosidl action type support data. Actions are build based on services
```

Comment:
Brief should only be one sentence. Add multiple lines if you want to have more descriptive comments.

```
/// Contains rosidl action type support data.
/**
 * Actions are built based on services...
 */ 
```